sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/AnalyzeHealthcareEntityAsyncClient.java

```diff
@@ -191,7 +197,7 @@ private PagedResponse<AnalyzeHealthcareEntitiesResultCollection> toTextAnalytics
                 analyzeHealthcareEntitiesResultCollection, batchStatistic);
         }
 
-        final List<TextAnalyticsError> errors = healthcareJobState.getErrors();
+        final List<Error> errors = healthcareJobState.getErrors();
```

Comment:
is this a public API change?

Reply:
There are two TextAnalyticsErrors, one is hidden in the /implementation folder and another one is publically explored. 
The delete one is the hidden one
https://github.com/Azure/azure-sdk-for-java/pull/28262/files/b64d475b6d8efb21a4696d543687525425f3c87f#diff-93372ec78c6c34d069a028c529595a7d5efde4f5267c09700b455e0e719f4a8eL17